app/cdap/components/ParsingDrawer/styles.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles(() => {
+  return {
+    parsingDrawerStyles: {
+      width: 460,
+      height: '100%',
+      padding: '5px 20px 5px 30px',
+    },
+    panelStyles: {
+      height: '100%',
+    },
+    paper: {
+      top: '46px',
+    },
+    headerStyles: {
+      height: '60px',
```

Comment:
use number directly instead of using number with px everywhere

Reply:
Done